artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/queue/UpdateQueue.java

```diff
@@ -31,7 +31,7 @@
    String name;
 
    @Option(name = "--deleteOnNoConsumers", description = "whether to delete when it's last consumers disconnects)")
-   Boolean deleteOnNoConsumers = null;
+   String deleteOnNoConsumers = null;
```

Comment:
@clebertsuconic You only get -1 like every one else ;) 

TBH I care less about the mechanism and more about the abiliity to update a queue.  If there's precedence for this case already set, then fair enough.  @franz1981 probably best to be consistent for the time being.  I would like to see more effort put into the CLI in general, IMO it needs some care and attention, but we can address that at a later date.